backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc

```diff
@@ -0,0 +1,279 @@
+
+* You have verified that only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
++
+[NOTE]
+====
+In {product-title} version 4.12 or later, verify that this is the only default `volumeSnapshotClass`.
+====
+
+* You have verified that `deletionPolicy` of the `VolumeSnapshotClass` CR is set to `Retain`.
+
+* You have verified that only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
+
+* You have included the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR.
+
+* You have verified that the `OADP namespace` has the annotation `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`.
++
+[NOTE]
+====
+In OADP 1.1 the above setting is mandatory.
+
+In OADP 1.2 the `privileged-movers` setting is not required in most scenarios. The restoring container permissions should be adequate for the Volsync copy. In some user scenarios, there may be permission errors that the `privileged-mover`= `true` setting should resolve.
+====
+
+* You have installed the VolSync Operator by using the Operator Lifecycle Manager (OLM).
++
+[NOTE]
+====
+The VolSync Operator is required for using OADP Data Mover.
+====
+
+* You have installed the OADP operator by using OLM.
+
+.Procedure
+
+. Configure a Restic secret by creating a `.yaml` file as following:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret_name>
+  namespace: openshift-adp
+type: Opaque
+stringData:
+  RESTIC_PASSWORD: <secure_restic_password>
+----
++
+[NOTE]
+====
+By default, the Operator looks for a secret named `dm-credential`. If you are using a different name, you need to specify the name through a Data Protection Application (DPA) CR using `dpa.spec.features.dataMover.credentialName`.
+====
+
+. Create a DPA CR similar to the following example. The default plugins include CSI.
++
+.Example Data Protection Application (DPA) CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <bucket_name>
+          prefix: <bucket-prefix>
+        provider: aws
+  configuration:
+    restic:
+      enable: <true_or_false>
+    velero:
+       itemOperationSyncFrequency: "10s"
+       defaultPlugins:
+        - openshift
+        - aws
+        - csi
+        - vsm <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14" <4>
+      volumeOptions: <5>
```

Comment:
Wrong configuration key and indentation.

correct one is: 

volumeOptionsForStorageClasses:
  sourceVolumeOptions:
    < ...... >
  destinationVolumeOptions:
    < ...... >

Reply:
@stillalearner See - https://github.com/openshift/openshift-docs/pull/62706